linting-configs/.ansible-lint

```diff
@@ -0,0 +1,50 @@
+---
+# .ansible-lint
+
+profile: moderate
+verbosity: 1
+strict: true
+
+# Enforce variable names to follow pattern below, in addition to Ansible own
+# requirements, like avoiding python identifiers. To disable add `var-naming`
+# to skip_list.
+var_naming_pattern: ^[a-zA-Z0-9_]*$
+
+use_default_rules: true
+
+# Ansible-lint is able to recognize and load skip rules stored inside
+# `.ansible-lint-ignore` (or `.config/ansible-lint-ignore.txt`) files.
+# To skip a rule just enter filename and tag, like "playbook.yml package-latest"
+# on a new line.
```

Comment:
I saw this [recommendation](https://ansible.readthedocs.io/projects/lint/configuring/#ansible-lint-configuration) to use the ignore file to ignore rules instead of the skip_list in .ansible-lint and wondered if we should consider using that approach for the stated reason?

```
# Ansible-lint is able to recognize and load skip rules stored inside
# `.ansible-lint-ignore` (or `.config/ansible-lint-ignore.txt`) files.
# To skip a rule just enter filename and tag, like "playbook.yml package-latest"
# on a new line.
# Optionally you can add comments after the tag, prefixed by "#". We discourage
# the use of skip_list below because that will hide violations from the output.
# When putting ignores inside the ignore file, they are marked as ignored, but
# still visible, making it easier to address later.
```

Reply:
I was going to go that route for the warnings I was going to fix in a future PR (command-instead-of-shell, command-instead-of-module, and deprecated-module. See comment above). Using the ignore rules ends up showing the warning with the string (ignored) next in the output workflow and it won't cause the linter to fail, which is useful.

However, GitHub will automatically parse all output from the linter run (even if we marked it as ignore) and overlay it over the the PR code changes making it hard to read/review. Given that I am planning to fix these warnings anyways in the next sprint I'm putting them under skip for now.

Reply:
I wondered if you had tried it already but thought I'd ask. Too bad that github overlays the ignored changes over the PR code since I think that just ignoring them not skipping them would be more valuable to us in the long run. I understand why you went with that decision though, thanks for the explanation.